README.md

```diff
@@ -21,50 +21,66 @@ This is an app that allows a customer to view a takeaway menu and place an order
 How to run
 -----
 Require the main application file in irb
-`2.5.0 :001 > require './lib/takeaway.rb'
- => true`
+```
+2.5.0 :001 > require './lib/takeaway.rb'
+ => true
+ ```
  Instantiate a takeaway object
-`2.5.0 :002 > t = Takeaway.new
- => #<Takeaway:0x00007fae351f8b10 @menu=#<Menu:0x00007fae351f8ae8 @dishes=[]>, @order_class=Order, @current_order=nil, @order_calculations=OrderCalculations>`
+```
+2.5.0 :002 > t = Takeaway.new
+ => #<Takeaway:0x00007fae351f8b10 @menu=#<Menu:0x00007fae351f8ae8 @dishes=[]>, @order_class=Order, @current_order=nil, @order_calculations=OrderCalculations>
+ ```
  Add some dishes to the menu
-`2.5.0 :003 > t.menu.add_dish(Dish.new("Pizza", 8))
+```2.5.0 :003 > t.menu.add_dish(Dish.new("Pizza", 8))
  => [#<Dish:0x00007fae35200b58 @name="Pizza", @price=8>]
 2.5.0 :004 > t.menu.add_dish(Dish.new("Pasta", 7))
  => [#<Dish:0x00007fae35200b58 @name="Pizza", @price=8>, #<Dish:0x00007fae3705d2e0 @name="Pasta", @price=7>]
 2.5.0 :005 > t.menu.add_dish(Dish.new("Salad", 6.50))
- => [#<Dish:0x00007fae35200b58 @name="Pizza", @price=8>, #<Dish:0x00007fae3705d2e0 @name="Pasta", @price=7>, #<Dish:0x00007fae381dcef8 @name="Salad", @price=6.5>]`
+ => [#<Dish:0x00007fae35200b58 @name="Pizza", @price=8>, #<Dish:0x00007fae3705d2e0 @name="Pasta", @price=7>, #<Dish:0x00007fae381dcef8 @name="Salad", @price=6.5>]
+ ```
  View the formatted menu
-`2.5.0 :006 > t.view_menu
+```
+2.5.0 :006 > t.view_menu
     Today's Menu
```

Comment:
Your menu is clearly presented - it shows you have thought about the user experience :)